packages/runtime/lib/app.js

```diff
@@ -98,9 +105,7 @@ class PlatformaticApp {
   }
 
   async stop () {
-    if (!this.#started) {
-      throw new Error('application has not been started')
-    }
+    if (!this.#started) return
```

Comment:
We want to start/stop individual services. So that means outside of the worker we can not be sure about the app's state. So either we check the app status each time before closing it or we don't throw an error. Is there a reason why this error throw is required?

Reply:
Throwing is not required, but I think it's a good sanity check. If someone keeps pressing the Stop button in a UI that's one thing, but if normal code is hitting this scenario, it could be a bug.

> So either we check the app status each time before closing

Definitely don't do that if you can avoid it. It's a race condition.

> or we don't throw an error

Is there anything wrong with handling the error and choosing to ignore it?

Reply:
Throwing is better here.